### ISSUE TITLE
There is some error happened when i used virtctl testvm vnc in second time

### ISSUE THREAD

**Issue by DeepLJH0001** [kubevirt/kubevirt#2269]
I am using xshell to ssh my server,  when i first time use virtctl vm vnc it's work. like this 
```
[root@localhost X11]#  virtctl vnc vm-pool-s3-76d79ffc8c-kj4tx
{"component":"virtctl","level":"info","msg":"connection timeout: 1m0s","pos":"vnc.go:137","timestamp":"2019-05-10T11:28:04.575477Z"}
{"component":"virtctl","level":"info","msg":"remote-viewer connected in 831.88047ms","pos":"vnc.go:148","timestamp":"2019-05-10T11:28:05.407359Z"}
Error encountered: read tcp 127.0.0.1:42194->127.0.0.1:49024: read: connection reset by peer
``` 
i close this connection and try second time , error happened，xmanager was crashed and exit immediately。
```
[root@localhost X11]#  virtctl vnc vm-pool-s3-76d79ffc8c-kj4tx --v=8
{"component":"virtctl","level":"info","msg":"Config loaded from file/root/.kube/config","pos":"loader.go:359","timestamp":"2019-05-10T11:35:58.249144Z"}
{"component":"virtctl","level":"info","msg":"GET wss://172.24.21.37:6443/apis/subresources.kubevirt.io/v1alpha3/namespaces/default/virtualmachineinstances/vm-pool-s3-76d79ffc8c-kj4tx/vnc","pos":"round_trippers.go:383","timestamp":"2019-05-10T11:35:58.250468Z"}
{"component":"virtctl","level":"info","msg":"Request Headers:","pos":"round_trippers.go:390","timestamp":"2019-05-10T11:35:58.250525Z"}
{"component":"virtctl","level":"info","msg":"    Authorization: Bearer kubeconfig-user-29nx7.c-mbvln:6krvlgbg9wfpq42rfqjljsh62hfbmk82r4r87dpt8jf5dpsmcp47w9","pos":"round_trippers.go:393","timestamp":"2019-05-10T11:35:58.250554Z"}
{"component":"virtctl","level":"info","msg":"    User-Agent: virtctl/v0.0.0 (linux/amd64) kubernetes/$Format","pos":"round_trippers.go:393","timestamp":"2019-05-10T11:35:58.250584Z"}
{"component":"virtctl","level":"info","msg":"connection timeout: 1m0s","pos":"vnc.go:137","timestamp":"2019-05-10T11:35:58.440525Z"}
{"component":"virtctl","level":"info","msg":"Executing commandline: 'remote-viewer [vnc://127.0.0.1:34039 --debug]'","pos":"vnc.go:225","timestamp":"2019-05-10T11:35:58.440710Z"}
{"component":"virtctl","level":"info","msg":"remote-viewer connected in 662.986001ms","pos":"vnc.go:148","timestamp":"2019-05-10T11:35:59.103512Z"}
Error encountered: websocket: close 1006 (abnormal closure): unexpected EOF
```

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**Environment**:
- KubeVirt version (use `virtctl version`):
```
Client Version: version.Info{GitVersion:"v0.15.0", GitCommit:"432f91f3a2bdb3c2624377ad327270c9e146d845", GitTreeState:"clean", BuildDate:"2019-03-05T08:21:47Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{GitVersion:"v0.15.0", GitCommit:"432f91f3a2bdb3c2624377ad327270c9e146d845", GitTreeState:"clean", BuildDate:"2019-03-05T09:55:30Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
```
- Kubernetes version (use `kubectl version`): rancher version 1.13.1
- VM or VMI specifications:
```
#the same like https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/vm.yaml
```
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release): centos 7.6
- Kernel (e.g. `uname -a`):
```
Linux localhost.localdomain 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux   
```
- Install tools:
xshell
- Others:

**Comment by slintes** [comment]
Hi @DeepLJH0001, I think this is fixed by https://github.com/kubevirt/kubevirt/pull/2171
Can you please update KubeVirt to the latest version (v0.17.0) and retry?

**Comment by DeepLJH0001** [comment]
> Hi @DeepLJH0001, I think this is fixed by #2171
> Can you please update KubeVirt to the latest version (v0.17.0) and retry?

it's work! thanks~：)

**Comment by slintes** [comment]
great :)

/close
